assets/static/querystring.js

```diff
@@ -39,9 +38,20 @@ var QueryString = (function() {
     }
 
 
+    remove = function(key) {
+        var baseUrl = [location.protocol, '//', location.host, location.pathname].join(''),
```

Comment:
I'm not a web developer so I don't know these things. To me it's weird that it should return it with the colon.